tests/common_setup.py

```diff
@@ -169,6 +170,9 @@ def multitenancy_setup_without_client(request):
                         -f %s up -d" % (conftest.mt_docker_compose_file),
                         use_common_files=False)
 
+    # wait a bit for the backend to start
+    time.sleep(30)
```

Comment:
I must reiterate. Don't use sleeps. This *will* bite us later, and is the reason our tests don't work on crappy droplets. I strongly urge you to *detect* that the server is up instead of relying on a sleep. It could have an eventual timeout of course, but this should be *much* longer.

Reply:
oops, yes, good point. Fixed

Reply:
I see the fix and I don't think it's complete. Even though the container is up, the service may still be starting up. On the other hand, I don't see us implementing proper synchronization anyway.

Reply:
so what do you suggest I do here?

Reply:
Ideally we should have some health check endpoint in each service. But here, I don't think we can do anything reasonable here without the changes in other components of the system.

Reply:
Maybe just try an endpoint that is supposed to give 401 Forbidden or something? Or some other expected failure.

Reply:
is there anything that can be done here?

Reply:
Just make a request to some random endpoint and make sure you get the expected response? (even if that response is an error)

Reply:
@kacf I checked with @bboozzoo and there is no way of doing this. 

What we currently have is wait for all the containers to be up, and after 30 seconds run tests. I don't know what else we can do here, without an endpoint verifying if containers are indeed up and running, and connected to their db